src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/DefaultTranscriptsList.tsx

```diff
@@ -45,29 +45,45 @@ import styles from './DefaultTranscriptsList.scss';
 type Props = {
   gene: Gene;
   rulerTicks: TicksAndScale;
-  expandedTranscriptIds: string[];
-  expandedTranscriptDownloadIds: string[];
-  toggleTranscriptInfo: (id: string) => void;
-  sortingRule: SortingRule;
 };
 
 const DefaultTranscriptslist = (props: Props) => {
-  const { gene, sortingRule } = props;
+  const expandedTranscriptIds = useSelector(getExpandedTranscriptIds);
+  const expandedTranscriptDownloadIds = useSelector(
+    getExpandedTranscriptDownloadIds
+  );
+  const sortingRule = useSelector(getSortingRule);
+  const filters = useSelector(getFilters);
+  const dispatch = useDispatch();
+
+  const { gene } = props;
 
   const sortingFunction = transcriptSortingFunctions[sortingRule];
   const sortedTranscripts = sortingFunction(gene.transcripts) as Transcript[];
 
   const [isFilterOpen, setFilterOpen] = useState(false);
 
   useEffect(() => {
-    const hasExpandedTranscripts = !!props.expandedTranscriptIds.length;
+    const hasExpandedTranscripts = !!expandedTranscriptIds.length;
 
     // Expand the first transcript by default
     if (!hasExpandedTranscripts) {
-      props.toggleTranscriptInfo(sortedTranscripts[0].stable_id);
+      dispatch(toggleTranscriptInfo(sortedTranscripts[0].stable_id));
     }
   }, []);
 
+  const shouldShowIndicator = () => {
+    if (sortingRule !== SortingRule.DEFAULT) {
+      return true;
+    }
+
+    if (Object.values(filters).some((filterValue) => filterValue)) {
```

Comment:
`.some` returns a boolean. Which means that starting from this line, the body of the function can be rewritten to:

`return Object.values(filters).some((filterValue) => filterValue)`

or `return Object.values(filters).some(Boolean)`

Reply:
✅

Reply:
<img src="https://user-images.githubusercontent.com/6834224/102719632-f8dbd580-42e6-11eb-9e83-ab992f84dbaf.png" width="500" />

I meant

```
  const shouldShowIndicator = () => {
    if (sortingRule !== SortingRule.DEFAULT) {
      return true;
    }

    return Object.values(filters).some(Boolean);
  };
```

or, even terser:

```
  const shouldShowIndicator = () =>
    sortingRule !== SortingRule.DEFAULT
      || Object.values(filters).some(Boolean);
```

Reply:
✅